src/ring_theory/noetherian.lean

```diff
@@ -243,16 +250,19 @@ theorem is_noetherian_iff_well_founded
       rw [← hs₂, sup_assoc, ← submodule.span_union], simp }
   end⟩
 
-def is_noetherian_ring (α) [ring α] : Prop := is_noetherian α α
+@[class] def is_noetherian_ring (α) [ring α] : Prop := is_noetherian α α
+
+instance is_neotherian_ring.to_is_neotherian {α : Type*} [ring α] :
+  ∀ [is_noetherian_ring α], is_noetherian α α := id
```

Comment:
neotherian -> noetherian x 2 on line 255

Reply:
I don't understand.

Reply:
You've misspelled noetherian twice.

Reply:
I see. I always do that.